From f3d2a6ca2d64df96cdee6d42b6f6871138a917d2 Mon Sep 17 00:00:00 2001
From: Daniel Kahn Gillmor <dkg@fifthhorseman.net>
Date: Fri, 24 Oct 2014 14:00:44 -0400
Subject: [PATCH] Add new lock-obj-pub for mips64el-unknown-linux-gnuabi64.

* src/syscfg/lock-obj-pub.mips64el-unknown-linux-gnuabi64.h: New.
* src/Makefile.am (lock_obj_pub): Add.

--

native lock obj header for 64-bit little-endian MIPS, taken from
debian mips64el port buildd, see:

http://mips64el.debian.net/debian/buildlog/libg/libgpg-error_1.16-2/libgpg-error_1.16-2_mips64el-20140928-1753.build

Debian-Bug-Id: 766135
---
 src/Makefile.am                                    |  1 +
 .../lock-obj-pub.mips64el-unknown-linux-gnuabi64.h | 25 ++++++++++++++++++++++
 2 files changed, 26 insertions(+)
 create mode 100644 src/syscfg/lock-obj-pub.mips64el-unknown-linux-gnuabi64.h

diff --git a/src/Makefile.am b/src/Makefile.am
index 903feae..b91876b 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -55,6 +55,7 @@ lock_obj_pub = \
         syscfg/lock-obj-pub.i586-pc-linux-gnu.h             \
         syscfg/lock-obj-pub.m68k-unknown-linux-gnu.h        \
         syscfg/lock-obj-pub.mips-unknown-linux-gnu.h        \
+        syscfg/lock-obj-pub.mips64el-unknown-linux-gnuabi64.h \
         syscfg/lock-obj-pub.mipsel-unknown-linux-gnu.h      \
         syscfg/lock-obj-pub.or1k-unknown-linux-gnu.h        \
         syscfg/lock-obj-pub.powerpc-unknown-linux-gnu.h     \
diff --git a/src/syscfg/lock-obj-pub.mips64el-unknown-linux-gnuabi64.h b/src/syscfg/lock-obj-pub.mips64el-unknown-linux-gnuabi64.h
new file mode 100644
index 0000000..8a81e3f
--- /dev/null
+++ b/src/syscfg/lock-obj-pub.mips64el-unknown-linux-gnuabi64.h
@@ -0,0 +1,25 @@
+## lock-obj-pub.mips64el-unknown-linux-gnuabi64.h
+## File created by gen-posix-lock-obj - DO NOT EDIT
+## To be included by mkheader into gpg-error.h
+
+typedef struct
+{
+  long _vers;
+  union {
+    volatile char _priv[40];
+    long _x_align;
+    long *_xp_align;
+  } u;
+} gpgrt_lock_t;
+
+#define GPGRT_LOCK_INITIALIZER {1,{{0,0,0,0,0,0,0,0, \
+                                    0,0,0,0,0,0,0,0, \
+                                    0,0,0,0,0,0,0,0, \
+                                    0,0,0,0,0,0,0,0, \
+                                    0,0,0,0,0,0,0,0}}}
+##
+## Local Variables:
+## mode: c
+## buffer-read-only: t
+## End:
+##
-- 
2.1.1

